### PR TITLE
Fixes for the Russian Translation

### LINKED ISSUE #567: Improve Russian Translation
According to @julepka the Russian Translation should be improved.

This can be done right away or wait for the refactoring to be finished (surely still a couple of months)

### PR DESCRIPTION
**Pull request by CthUlhUzzz**
Corrected typos in the Russian translation. Translated the latest changelog update.

Closes #567

### CODE REVIEW

**Comment by cpholguera** [comment]
Hi @CthUlhUzzz, thanks a lot for your PR addressing that issue. We'll try to review it as soon as possible.

@julepka if you have some time could you please take a look? Thanks a lot in advance ☺️

File: Document-ru/CHANGELOG.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
 
 ## V1.3 - 13 May 2021
 
-We are proud to announce the introduction of a new document build pipeline, which is a major milestone for our project. The build pipeline is based on [Pandocker](https://github.com/dalibo/pandocker) and [Github Actions](https://github.com/OWASP/owasp-masvs/tree/master/.github/workflows).
-This significantly reduces the time spent on creating new releases and will also be the foundation for the OWASP MSTG and will be made available for the OWASP ASVS project.
+Мы с гордостью сообщаем о введении новой системы генерации документов, которая является важной вехой для нашего проекта. Система генерации основана на проектах [Pandocker](https://github.com/dalibo/pandocker) и [Github Actions](https://github.com/OWASP/owasp-masvs/tree/master/.github/workflows).
+Нововведение сокращает время, затрачиваемое на создание новых выпусков, а также станет основой для OWASP MSTG и будет также доступно для проекта OWASP ASVS.
```
Comment (julepka) [inline Document-ru/CHANGELOG.md:6]:
```suggestion
Нововведение значительно сокращает время, затрачиваемое на создание новых сборок, а также станет основой для OWASP MSTG и будет доступно для проекта OWASP ASVS.
```

File: Document-ru/CHANGELOG.md [hunk 2]
```diff
@@ -2,19 +2,19 @@
 
 ## V1.3 - 13 May 2021
 
-We are proud to announce the introduction of a new document build pipeline, which is a major milestone for our project. The build pipeline is based on [Pandocker](https://github.com/dalibo/pandocker) and [Github Actions](https://github.com/OWASP/owasp-masvs/tree/master/.github/workflows).
-This significantly reduces the time spent on creating new releases and will also be the foundation for the OWASP MSTG and will be made available for the OWASP ASVS project.
+Мы с гордостью сообщаем о введении новой системы генерации документов, которая является важной вехой для нашего проекта. Система генерации основана на проектах [Pandocker](https://github.com/dalibo/pandocker) и [Github Actions](https://github.com/OWASP/owasp-masvs/tree/master/.github/workflows).
```
Comment (julepka) [inline Document-ru/CHANGELOG.md:5]:
```suggestion
Мы с гордостью сообщаем о введении нового конвейера сборки документов, который является важной вехой для нашего проекта. Конвейер сборки основан на проектах [Pandocker](https://github.com/dalibo/pandocker) и [Github Actions](https://github.com/OWASP/owasp-masvs/tree/master/.github/workflows).
```

`build` is usually translated as `сборка`. I've found that Microsoft uses `конвейер сборки` in their docs and, as for me, it sounds more understandable: https://docs.microsoft.com/ru-ru/learn/modules/create-a-build-pipeline/

**Review by julepka — COMMENTED** [review]
@CthUlhUzzz good job!
I've added a couple of suggestions. Please, review.

File: Document-ru/CHANGELOG.md [hunk 3]
```diff
@@ -2,19 +2,19 @@
...
+- Добавлено требование MSTG-PLATFORM-11
 
-### Special Thanks
+### Специальная благодарность
 
-- Jeroen Willemsen for kick-starting this initiative last year!
-- Damien Clochard and Dalibo for supporting and professionalizing the build pipeline.
-- All our Hindi, Farsi, Portuguese and Brazilian Portuguese collaborators for the excellent translation work.
+- Jeroen Willemsen за запуск инициативы в прошлом году!
+- Damien Clochard и Dalibo за поддержку и профессионализацию системы генерации.
```
Comment (julepka) [inline Document-ru/CHANGELOG.md:16]:
```suggestion
- Damien Clochard и Dalibo за поддержку и профессионализацию конвейера сборки.
```

**Comment by CthUlhUzzz** [comment]
@julepka, Done 👍

**Review by julepka — APPROVED** [review]

**Comment by cpholguera** [comment]
Thanks a lot @CthUlhUzzz and @julepka for the corrections, this is very helpful!

**Comment by cpholguera** [comment]
Hi @CthUlhUzzz, we need one last thing before merging. Could you please pull the latest changes from master? Thanks!

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork

**Comment by CthUlhUzzz** [comment]
@cpholguera, I'm ready 😎

**Review by cpholguera — APPROVED** [review]
Thanks a lot @CthUlhUzzz we're ready to go! and btw. since this is your first contribution I officially welcome you to the team. You'll appear as a new contributor on our next MASVS release!

We'll be happy to have more contributions from you, please take a look at the MSTG open Issues and Discussions or ask us directly and we can gladly guide you.

**Review by cpholguera — APPROVED** [review]
Thanks a lot @CthUlhUzzz we're ready to go! and btw. since this is your first contribution I officially welcome you to the team. You'll appear as a new contributor on our next MASVS release!

We'll be happy to have more contributions from you, please take a look at the MSTG open Issues and Discussions or ask us directly and we can gladly guide you.

**Pull request merged**